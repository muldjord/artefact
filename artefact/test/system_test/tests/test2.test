INPUT="queryonelm600p.xml"
EXPECT="test2_expected.xml"
FILTER="cat"
TITLE="Query one data entry"
