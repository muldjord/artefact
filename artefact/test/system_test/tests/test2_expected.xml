<?xml version='1.0' encoding='utf-8'?>
<results>
  <result id="0" errcode="0" errstring="" class="lm600p"
          deviceid="testid" devicetype="lm600p" timestamp="">
    <group name="lm600p">
      <group name="odxt">
        <value name="sph">-6.00</value>
        <value name="cyl">0.00</value>
        <value name="axis">0</value>
      </group>
      <group name="osin">
        <value name="sph">-6.00</value>
        <value name="cyl">0.00</value>
        <value name="axis">0</value>
      </group>
    </group>
  </result>
</results>
